### ISSUE TITLE
Not enough slick clone slides causing flashing during sliding between slider head & tail

### ISSUE THREAD

**Issue by jackytwh** [akiran/react-slick#557]
Glad if anyone can help, thanks a lot.

```
var settings = {
    centerMode: true,
    centerPadding: '100px'
}
```

https://jsfiddle.net/jackytwh/20bumb4g/1644/

**Comment by MikeJSX** [comment]
I created [PR](https://github.com/akiran/react-slick/pull/565) for this issue. Check if you'd like

**Comment by jackytwh** [comment]
@MikeJSX , thanks for looking into the issue.
However, the infinite value is default to be "YES" though i didn't state it.
Therefore, i wonder if you are fixing the code in the right place.
The issue seems still exist. Kindly take a look sample code in jsfiddle for details.

**Comment by szimek** [comment]
@MikeJSX For me your fix almost works 😉  The number of slides is correct now, but with centerMode + infinity I'm getting invalid offset for the whole track, i.e. the center slide is not marked as center, but the one on the right. Looking at the source code of the original slick carousel it seems one needs to adjust track left offset by additionally subtracting slide width, i.e. change [this](https://github.com/akiran/react-slick/blob/master/src/mixins/trackHelper.js#L118) line to:
```js
slideOffset += spec.slideWidth * Math.floor(spec.slidesToShow / 2) -  spec.slideWidth;
```
Now everything seems to be fine, at least in my particular case.

**Comment by laveesingh** [comment]
I am not able to reproduce this anymore. Maybe it got fixed around 8feb5cd. 
If someone is still able to regenerate it, please feel free to suggest reopening the issue.
